tests/helper/component_podman.go

```diff
@@ -110,6 +111,11 @@ func (o *PodmanComponent) GetPodDef() *corev1.Pod {
 	return &pod
 }
 
+func (o *PodmanComponent) GetJobDef() *batchv1.Job {
+	// Not implemented for Podman
+	return nil
```

Comment:
Maybe we could just `panic` right away to fail earlier?

```suggestion
	panic("not implemented for Podman")
```